tools/RunCredScan.ps1

```diff
@@ -0,0 +1,54 @@
+﻿$credentialScannerLocation = $args[0]+"\CredentialScanner.exe"
```

Comment:
Can we put a comment at the top of the file with an example of usage with all parameters?